src/graph/backend/dnnl/kernels/kernel_base.cpp

```diff
@@ -43,6 +43,24 @@ bool kernel_base_t::enabled_constant_cache() const {
     return enabled;
 }
 
+size_t kernel_base_t::encode_constant_cache_key(
+        const std::vector<tensor_t> &inputs, size_t cache_key) const {
+    // Encode the constant memory address into cache key for differentiation
+    size_t encoded_cache_key = cache_key;
+    for (const auto &in : inputs) {
+        if (in.get_logical_tensor().property
+                == dnnl_graph_tensor_property_t::
+                        dnnl_graph_tensor_property_constant) {
+            auto data_handle = in.get_data_handle();
+            if (data_handle != nullptr) {
```

Comment:
```suggestion
            if (data_handle) {
```
why do we need to check it?

Reply:
Actually no need to, removed~